libmozevent/mercurial.py

```diff
@@ -404,8 +404,9 @@ def is_eligible_for_retry(self, error):
         eligible_errors = [
             "push failed on remote",
             "stream ended unexpectedly",
-            "error: EOF occurred in violation of protocol",
+            "error: eof occurred in violation of protocol",
```

Comment:
@abpostelnicu sorry I did not catch this in the review of #102 :(

Reply:
I changed it in 84664f263bf86aca3b0a85ec3d3d81feb27e78a1 to be done automatically.